zeppelin-server/notebook/python.recovery

```diff
@@ -0,0 +1 @@
+2CZA1DVUG:shared_process	192.168.3.2:55410
```

Comment:
should this be in the commit? and recovery.crc?

Reply:
Will delete it